app/assets/javascripts/controllers/report_data_controller.js

```diff
@@ -341,10 +341,14 @@
     if (mainContent) {
       angular.element(mainContent).removeClass('miq-sand-paper');
       angular.element(mainContent).removeClass('miq-list-content');
+      angular.element(document.querySelector('#paging_div .miq-pagination')).css('display', 'none');
+      console.log(document.getElementById('paging_div'));
```

Comment:
Can you please remove `console.log`? Not sure why it is not part of eslint rules.